Reaktoro/Transport/TransportSolver.hpp

```diff
@@ -204,7 +200,7 @@ class Mesh
 };
 
 /// A class for solving advection-diffusion problem.
-/// Eq: du/dt + v*du/dx = D*d�u/dx�
+/// Eq: du/dt + v*du/dx = D*d�u/dx�
```

Comment:
```suggestion
/// Eq: du/dt + v*du/dx = D*d²u/dx²
```
You may want to change your text editor to UTF-8 encoding.